### PR TITLE
Ian - tutorial template

### PR DESCRIPTION
**Pull request by fineon**
> __ALL Pull-Requests require an associated [Issue](https://github.com/thegooddocsproject/templates/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) as documented in [How to contribute](https://github.com/thegooddocsproject/templates/blob/master/CONTRIBUTING.md#contributing).__

## Purpose / why

I'm adding more information for the tutorial template to adapt to current base templates and current works by other members. I also added an example of my own, as a web developer who have followed other technical documentations and hope that we can understand a bit more about how developer learn and document projects.

## What changes were made?

Added a new example file for the tutorial template called `tutorial-template-example.md` that explains an example technical project that I made up and applies the rules of the tutorial template. 

I also added more information about last modified date and document version for the sake of clarity when we have members going back and forth editing the templates. 

## Verification

- Check for spelling mistakes, sentences that don't match voice and brand guidelines.
- For the example tutorial, check for correct usage of technical words in software development and syntax highlights. 

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Are issues linked correctly?
* [ ] Is this PR labeled correctly?
* [ ] If template updates: do they align with [developers.google.com/style/](https://developers.google.com/style/)?
* [ ] Did the PR receive at least one :+1: and no :-1: from core-maintainers?
* [ ] On merging, did you complete the merge using [keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue)?
* [ ] On merging, did you add any applicable notes to a [draft release](https://github.com/thegooddocsproject/templates/releases) and link to this PR?

### CODE REVIEW

**Comment by camerons** [comment]
@fineon I haven't had a chance to review yet, but ask that you change this pull request to be against the "dev" branch instead of the main branch. At the moment, our main branch is frozen, and we are developing these template improvements on the dev branch.

**Comment by fineon** [comment]
Changed the PR to merge into `dev` branch now. Thanks for letting me know Cameron

File: tutorial/tutorial-template-example.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+**Last updated:** 2021-04-05 (YYY-MM-DD) by Ian
+
+---
+# Fictional example project
+
+Here's an example of a small open source project that aims to guide developers and users through how to use their library
+
+# An example of a technical project
+Octopus library offers a great toolset to build a model of an octopus on web browser, from tentacle movements to the color and textures of an octopus. We want to guide developers and users through how they can use the library to build an octopus using Javascript, CSS and HTML.
```
Comment (radisysimllc) [inline tutorial/tutorial-template-example.md:27]: The Octopus library

**Comment by radisysimllc** [comment]
@fineon Lan, this is a really good sample tutorial! You'll notice I've added several suggested changes to the instructional text (not touching the code samples) as I went through the tutorial sample based on the tutorial template.

**Review by radisysimllc — CHANGES_REQUESTED** [review]
@fineon Lan, this is a really good sample tutorial! 

You'll notice I've added several suggested changes to the instructional text (not touching the code samples) as I went through the tutorial sample.

**Comment by barbaricyawps** [comment]
@fineon , I love that you've taken the initiative to provide a tutorial example. That's so cool! FYI, our base template is still in a little bit of flux, but I really like what I see here. I agree with many of @radisysimllc 's suggestions too. Nice work here!

I don't see any immediate blockers for merging this in at this point, but I wonder if you'd be willing to be patient for a few weeks while we're refining our policies and procedures for submitting new templates and template improvements just in case it could impact your PR. I promise we're actively working on streamlining our contribution processes!

**Comment by fineon** [comment]
No worries Alyssa! Hoping to hear an update soon!

**Comment by barbaricyawps** [comment]
@fineon , we've made significant changes to the tutorial template and so your PR is unfortunately out of date now. But the great news is that you'll be able to build an awesome tutorial template on the Chronologue project one day! I'm going go close this PR for now.

**Pull request closed without merging**